### PR TITLE
DOC: Clarify TETE description

### LINKED ISSUE #11109: Minor clarifications needed in TETE documentation.
While the documentation for the new TETE frame is excellent and quite thorough, a couple of minor clarifications would be helpful.

> TETE is the apparent coordinate system used by JPL Horizons and is the correct coordinate system to use when combining the right ascension with local apparent sidereal time to calculate the hour angle.

Hour angle here should be "apparent" or "TIRS" hour angle to avoid giving the false impression that this is the observed (or ITRS) hour angle which will only be available when an observed `HADec` frame is implemented (see issue #10239).

> This frame also includes frames that are defined relative to the Earth, but that are offset (in both position and velocity) from the Earth.

Just saying "the Earth" here is a bit misleading. The Moon is offset from the Earth, but an observer's position on the Moon could not be appropriately defined in TETE. Center of the Earth (or geocenter) would be better and the offset should be described as "from the center of the Earth (or geocenter) in the ITRS".

### PR DESCRIPTION
**Pull request by NikitaTewary**
Fixes #11109 

### Description

Two small changes done in order to give some clarity in the TETE Documentation.
The following image shows the changes that i made.

EDIT: Formatting

### CODE REVIEW

**Review by astrojuanlu — APPROVED** [review]
Thanks @NikitaTewary for contributing to Astropy! 🎉 

This looks good to me, but pinging @mkbrewer, @mhvk and @StuartLittlefair for a second pair of eyes just in case. I don't think a changelog entry is needed.

**Comment by NikitaTewary** [comment]
Thank you, for your time.
I am new to this open-source culture, i am still learning everything, i
have skill set of python and have starting learning CSS simultaneously,
thats why i find this project apt for me, i want to contribute soul
heartedly to this project.
If given a change i would love to learn and contribute under your guidance.
Best regards
Nikita Tewary
2nd Year B.Tech student

File: astropy/coordinates/builtin_frames/equatorial.py
Comment (mkbrewer) [inline astropy/coordinates/builtin_frames/equatorial.py:64]: I'd like to see:
```
with local apparent sidereal time to calculate the apparent (TIRS) hour angle
```
here. While in the context of TETE, this is the apparent hour angle, it is the same as the hour angle that would be computed in the CIRS frame, so it's generally the TIRS hour angle.

Comment (NikitaTewary) [reply astropy/coordinates/builtin_frames/equatorial.py:64]: I am on it

Comment (NikitaTewary) [reply astropy/coordinates/builtin_frames/equatorial.py]: I have added (TIRS) at line 64, please review it.

**Comment by astrojuanlu** [comment]
@NikitaTewary One comment: I see your commit messages are `Mycontributiongiven` - please add a sentence in English describing the changes made in the commit, see https://docs.astropy.org/en/stable/development/workflow/git_edit_workflow_examples.html#commit-your-change for an example.

**Comment by pllim** [comment]
Hello and thank you for your contribution! I cancelled Actions CI just now for this PR, please do not be alarmed.

p.s. Would also be nice that the commits be squashed into one commit before merge, once all the comments have been addressed.

**Comment by NikitaTewary** [comment]
Okay

**Review by pllim — CHANGES_REQUESTED** [review]
There is a merge commit in the history now, please squash all the commits into one. Perhaps you might find https://docs.astropy.org/en/latest/development/workflow/development_workflow.html#how-to-squash helpful.

Also, you can add "[ci skip]" in your final commit message to skip CircleCI, as it is unnecessary.

Thank you!

**Review by astrojuanlu — CHANGES_REQUESTED** [review]
Hello @NikitaTewary, I see that you force pushed your branch in an attempt to rebase your changes and linearize your history (good!). However, your branch ended up containing commits from other people. This is normal at the beginning, git is hard and we've all been there :)

Notice that your branch now departs from an older version of `master` in Astropy, as you can see here:

https://github.com/astropy/astropy/network

To solve this, you will have to do a rebase. Steps:

1. Double check your setup https://docs.astropy.org/en/stable/development/workflow/development_workflow.html#double-check-your-setup
2. If necessary, add a second remote https://docs.astropy.org/en/stable/development/workflow/get_devel_version.html#tell-git-where-to-look-for-changes-in-the-development-version-of-astropy
3. Rebase https://docs.astropy.org/en/stable/development/workflow/development_workflow.html#how-to-rebase ℹ️ After this step, you should only have two commits ℹ️ 
4. Force push https://docs.astropy.org/en/stable/development/workflow/development_workflow.html#how-to-push
5. And for perfection, squash your commits into one https://docs.astropy.org/en/stable/development/workflow/development_workflow.html#how-to-squash and force push again

Let us know if you need help.

**Comment by pllim** [comment]
There is still one unrelated commit.

**Comment by astrojuanlu** [comment]
The commit message leads one to think that f282693cc does not belong here, but in fact it does. Not sure why it ended up like that though.

@NikitaTewary you followed the instructions I gave perfectly, thanks! But @pllim has a point, these two commits should indeed be one. For this, the best is to do a "squash on interactive rebase":

https://docs.astropy.org/en/stable/development/workflow/development_workflow.html#how-to-squash

which will present you this output:

```
pick f282693cc TST: Older pytest missing PytestUnhandledThreadExceptionWarning
pick a58c075be TIRS added for more clarity in the documentaion
```

and so you have to change it by this:

```
pick f282693cc TST: Older pytest missing PytestUnhandledThreadExceptionWarning
squash a58c075be TIRS added for more clarity in the documentaion
```

And then pick a final commit message (no need to write `Mycontributiongiven` at the end, and beware of typos though).

After this change, this is ready to merge.

**Comment by pllim** [comment]
@astrojuanlu , `TST: Older pytest missing PytestUnhandledThreadExceptionWarning` is actually from #11167 . It should not be here at all. FYI.

**Comment by astrojuanlu** [comment]
Yes, the message `TST: Older pytest missing PytestUnhandledThreadExceptionWarning` corresponds to 32e15438b48, but even if f282693 contains the same message (not sure why) the diff is actually corresponding to this change, that's why I say @NikitaTewary needs to squash them.

**Comment by mkbrewer** [comment]
Sorry that my request for a minor change caused so much trouble here.

**Comment by astrojuanlu** [comment]
No fuss @mkbrewer ! These small requests are the ones that are more palatable for newcomers. The back-and-forth with git is absolutely normal, I apologize if we made too much noise (if there is a specific channel in the Slack/Matrix chat for these kind of exchanges and it's preferred that they're moved there, please let me know)

**Comment by astrojuanlu** [comment]
Last thing @NikitaTewary, I promise: the first line of the commit message is still misleading, please do

```
$ git commit --amend "TIRS added for more clarity in the documentation"
```

and force push again.

**Comment by NikitaTewary** [comment]
@astrojuanlu @mkbrewer @pllim thank you for being so patient. I am a newcomer and I learnt a lot of new commands in git because of your directions, I just have one request if you could assign me more docs related issues I would love to fix it.
Thank you!

**Review by astrojuanlu — APPROVED** [review]
This looks good to me now!

**Review by pllim — APPROVED** [review]
The PR is in good shape now. However the commit author in the history is a little misleading because of the squash and what-nots. If you don't mind that being messed up, feel free to merge. Otherwise, it needs a `git commit --amend` to change the authorship. 🤷

**Comment by NikitaTewary** [comment]
@pllim what should be the desired Author name? My name or something else?

**Comment by pllim** [comment]
It currently shows me and you both as author, but in reality you are the author.

**Comment by saimn** [comment]
```
git commit --amend --author="Author Name <email@address.com>" --no-edit
```
where you put your name and email to get the proper credit.

**Comment by NikitaTewary** [comment]
C:\Users\Asus\OneDrive\Desktop\Astropy\astropy>git push -forced
fatal: the receiving end does not support push options
fatal: the remote end hung up unexpectedly
error: failed to push some refs to 'https://github.com/NikitaTewary/astropy.git'

Now, this is what is popping up whenever I am giving the push command, after I tried to change the Author

**Comment by astrojuanlu** [comment]
Weird! It was working for you before, right? I think it's a typo in `--forced`, (it should rather be `--force`, without `d`)

**Comment by astrojuanlu** [comment]
This is perfect now. Thanks everyone for the patience and congratulations @NikitaTewary for your first contribution 🎉

**Pull request merged**
